src/screens/Menu/Menu.js

```diff
@@ -196,6 +197,13 @@ const Menu = ({
       card: true,
       action: () => navigation.navigate(APP_SETTINGS),
     },
+    {
+      key: 'addressBook',
+      title: t('title.addressBook'),
```

Comment:
It's better to place this identifier next to others in settingsContent.settingsItem object. What do you think?

Reply:
Cool, I added it before other menu items were translated, will change! 👍